### ISSUE TITLE
Add copier question to fail tests on log WARNING

### ISSUE THREAD

**Issue by sbidoul** [OCA/oca-addons-repo-template#120]
Currently, tests pass even if there are WARNING in the Odoo log.

Previously we had a notice of that, via runbot, although it never prevented merging.
Since runbot will be shutdown, we are going to lack this notice, unless people look at the test logs.

Since many repos have WARNING, this check must be enabled per-repo (and maybe made mandatory on newer branches: 15, 16...).

Blocked on https://github.com/OCA/oca-ci/issues/10

**Comment by sbidoul** [comment]
It would be nice to have it before we open the 18.0 branches.
